recipe/meta.yaml

```diff
@@ -19,42 +19,49 @@ build:
   skip: true  # [win and vc<14]
   missing_dso_whitelist:  # [linux and s390x]
     - $RPATH/ld64.so.1    # [linux and s390x]
+  script_env:
+    # "Environment variable VCPKG_FORCE_SYSTEM_BINARIES must be set on
+    # arm, s390x, ppc64le and riscv platforms."
+    - VCPKG_FORCE_SYSTEM_BINARIES=1  # [ arm64 or aarch64 or s390x or ppc64le]
+
 requirements:
   build:
-    - make  # [not win]
-    - git   # [not win]
-    - sed   # [not win]
-    - gawk  # [not win]
-    - cmake <=3.21
-    - pkg-config  # [not win]
+    - m2-patch  # [win]
+    - ninja-base
+    - sed                    # [not win]
+    - gawk                   # [not win]
+    - cmake                  # >=3.21
```

Comment:
I would either remove the comment or constrain this properly with `cmake >=3.21` if there is reason to do so.